hs_core/models.py

```diff
@@ -2506,9 +2506,8 @@ def set_quota_holder(self, setter, new_holder):
         validate_user_quota(new_holder, self.size)
         if self.quota_holder:
             # ensure the new holder has a bucket, buckets only exist for users with resources
-            if not new_holder.userprofile._bucket_name:
-                istorage = self.get_irods_storage()
-                istorage.create_bucket(new_holder.userprofile.bucket_name)
+            istorage = self.get_irods_storage()
+            istorage.create_bucket(new_holder.userprofile.bucket_name)
```

Comment:
@sblack-usu one question for you: will this behave okay if the bucket already exists?

Reply:
yes, it is fine if the bucket already exists